latch_cli/snakemake/config/parser.py

```diff
@@ -134,12 +134,7 @@ def generate_metadata(
             fg="yellow",
         )
 
-    if not metadata_path.exists() and click.confirm(
```

Comment:
removing this because from user tests I've found that it is quite confusing. I think if the file doesn't exist its generally pretty safe to create it without user confirmation since we aren't overwriting anything.